src/kademlia/dht_storage.cpp

```diff
@@ -119,9 +119,18 @@ namespace
 		std::memcpy(item.value.get(), buf.data(), size);
 	}
 
+	system_clock::time_point clock_now()
+	{
+		static auto const stime = system_clock::now();
+		static auto const ctime = aux::time_now();
+
+		auto const secs = total_seconds(aux::time_now() - ctime);
+		return stime + seconds(secs);
```

Comment:
Yes, that's the reason. Now, The type `dht_immutable_item` will be promoted to a public place in a latter commit. I think it's better and more clear to have the type `system_clock::time_point` for `last_seen`. What do you think?